.github/workflows/doc-pr-build.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 
 jobs:
   build_documentation:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Is `24.04` not supported yet?

Reply:
No, we had the same problem last week on optimum, see [here](https://github.com/huggingface/optimum/pull/2050). Ubuntu latest comes with python 12, not supported by xla yet.